snap/snapcraft.yaml

```diff
@@ -91,7 +83,7 @@ parts:
 
       CPU_COUNT=$(lscpu --parse=CPU | grep -v "^#" | wc -l)
 
-      export DESTDIR=${SNAPCRAFT_PART_INSTALL}
+      export DESTDIR="$(readlink --canonicalize ${SNAPCRAFT_PART_INSTALL})"
```

Comment:
Is this necessary? Under normal circumstances the part install directory shouldn't include symlinks.

Reply:
Let me revisit that. When I tried that the path (or some component) ended up being relative which confused the `install` binary.

Reply:
It wouldn't hurt, it's just that it's not very common to use it but it will work.

Reply:
I agree. I checked this again and as you said the path is absolute.